docs/core/compatibility/sdk/10.0/nu1510-pruned-references.md

```diff
@@ -1,29 +1,29 @@
 ---
 title: "Breaking change - NU1510 is raised for direct references pruned by NuGet"
 description: "Learn about the breaking change in the .NET 10 SDK where NU1510 is raised for unnecessary direct package references."
-ms.date: 08/11/2025
+ms.date: 09/04/2025
 ai-usage: ai-assisted
 ms.custom: https://github.com/dotnet/docs/issues/45462
 ---
 
 # NU1510 is raised for direct references pruned by NuGet
 
-Starting in .NET 10, NuGet raises a [`NU1510` warning](/nuget/reference/errors-and-warnings/nu1510) when a project includes a direct package reference that overlaps with a framework-provided library and isn't required.
+Starting in the .NET 10 SDK, pruning is enabled by default for projects that target or multi-target .NET 10. When pruning is enabled, NuGet raises a [`NU1510` warning](/nuget/reference/errors-and-warnings/nu1510) when a project includes a direct package reference that overlaps with a framework-provided library and isn't required.
```

Comment:
```suggestion
Starting in the .NET 10 SDK, pruning is enabled by default for projects that target or multi-target .NET 10. When pruning is enabled and the project targets or multi-targets .NET 10, NuGet raises a [`NU1510` warning](/nuget/reference/errors-and-warnings/nu1510) when a project includes a direct package reference that overlaps with a framework-provided library and isn't required.
```

Gist is warning is for .NET 10 only, even if pruning is manually enabled for lower versions.